uptime-kuma/rootfs/etc/s6-overlay/s6-rc.d/uptime-kuma/run

```diff
@@ -10,4 +10,7 @@ cd /opt/uptime-kuma || bashio::exit.nok "Could not change directory to Uptime Ku
 
 bashio::log.info "Starting Uptime Kuma..."
 
-exec node server/server.js --data-dir="/data/"
+export UPTIME_KUMA_DB_TYPE=sqlite
+export UPTIME_KUMA_DB_NAME=kuma.db
```

Comment:
These two are required as they changed the way the DB is setup on master https://github.com/louislam/uptime-kuma/pull/2738/files#diff-c3ca73872fa7aee135c0942cff2eab343c8137fdb5b1d72a7d41ef9effbf9f16R49